tokenserver/src/middleware/ErrorHandlingMiddleware.js

```diff
@@ -2,8 +2,8 @@
 
 const errorHandlingMiddleware = (err, req, res, next) => {
   if (err.status != null) {
-    res.status(err.statusCode).json({
-        msg: err.message,
+    res.status(err.status).json({
+        msg: err.response.data.message,
```

Comment:
Not yet used?

Reply:
I don't understand the question? This is where the error message is in an official response from GitHub...

Reply:
In fairness, it makes little difference atm other than for debugging purposes, as the platform file handler immediately ignores the message.

Reply:
So we could in principle simplify to `"File not found"` or something similar